package.json

```diff
@@ -1,76 +1,89 @@
 {
-  "name": "@storyblok/vue-ws",
-  "private": true,
-  "version": "0.0.1",
+  "name": "@storyblok/vue",
+  "type": "module",
+  "version": "8.1.6",
+  "packageManager": "pnpm@9.12.2",
+  "description": "SDK to integrate Storyblok into your project using Vue.",
+  "author": "Storyblok",
   "license": "MIT",
-  "workspaces": [
-    "./lib",
-    "./playground"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storyblok/storyblok-vue"
+  },
+  "bugs": {
+    "url": "https://github.com/storyblok/storyblok-vue/issues"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./rsc": {
+      "types": "./dist/rsc/index.d.ts",
+      "import": "./dist/rsc.mjs",
+      "require": "./dist/rsc.js"
+    }
+  },
+  "main": "./dist/storyblok-vue.js",
+  "module": "./dist/storyblok-vue.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
   ],
-  "types": "./dist/types.index.d.ts",
   "scripts": {
-    "dev": "npm run dev --workspace=lib & npm run demo",
-    "demo": "npm run demo --workspace=playground",
-    "demo-vca": "npm run demo --workspace=playground-vca",
-    "build": "npm run build --workspace=lib",
-    "build:ts": "vite build && tsc --prokect tsconfigjson",
-    "test:unit": "npm run test:unit --workspace=lib",
-    "test:e2e": "npm run test:e2e --workspace=lib",
-    "test:components": "npm run test:components --workspace=lib",
-    "prepare": "husky install",
-    "lint": "eslint ."
+    "dev": "vite build --watch",
+    "playground": "pnpm run --filter ./playground/vue dev",
+    "build": "vite build ",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "cy:run": "cypress run",
+    "cy:open": "cypress open",
+    "cy:components": "cypress run-ct"
```

Comment:
we are missing a script for test running

Reply:
ok the dedicated test kinda of script right? let me add it